test/e2e.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# run e2e tests on must-gather-clean
+
+set -xeuo pipefail
+
+current_dir="$(dirname "${BASH_SOURCE[0]}" )"
+target_dir="${current_dir}/files"
+test_dir="${target_dir}/testrun-${RANDOM}"
+mkdir ${test_dir}
+config="examples/openshift_default.yaml"
```

Comment:
I updated the config today to also exclude more configmaps, I think the reports will not match anymore. Could you rebase please?